bytecomp/simplif.ml

```diff
@@ -566,7 +566,7 @@ let rec emit_tail_infos is_tail lambda =
   | Lletrec (bindings, body) ->
       List.iter (fun (_, lam) -> emit_tail_infos false lam) bindings;
       emit_tail_infos is_tail body
-  | Lprim (Pidentity, [arg]) ->
+  | Lprim ((Pidentity | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
@hhugo actually, if I remove this, it might introduce some performance regression here, I don't have strong opinions, I would like to hear other people's opinion